includes/definitions/discovery/lambdatrail.yaml

```diff
@@ -0,0 +1,299 @@
+mib: LAMBDATRAIL-MIB:Lambdatrail2s-MIB
+modules:
+    os:
+            hardware: .1.3.6.1.4.1.35616.5.1.2.0
+            serial: .1.3.6.1.4.1.35616.5.1.5.0
+            version: .1.3.6.1.4.1.35616.5.1.10.0
+            version_regex: '/(?<version>\S+),.*/'
+    sensors: 
+        temperature:
+            data:
+                -
+                    oid: lt2sMgmtPortTable
+                    value: lt2sMgmtSfpLaserTemp
+                    num_oid: '.1.3.6.1.4.1.35616.5.1.26.1.15.{{ $index }}'
+                    descr: Mgmt {{ $index }} Temp
+                    index: 'lt2sMgmtSfpLaserTemp.{{ $index }}'
+                    skip_values:
+                        -
+                            oid: lt2sMgmtSfpLaserTemp
+                            op: '='
+                            value: 0
+                -
+                    oid: lt2sPortTable
+                    value: lt2sLineLaserTemp
+                    num_oid: '.1.3.6.1.4.1.35616.5.3.1.1.15.{{ $index }}'
+                    descr: Line {{ $index }} Temp
+                    index: 'lt2sLineLaserTemp.{{ $index }}'
+                -
+                    oid: lt2sPortTable
+                    value: lt2sClientLaserTemp
+                    num_oid: '.1.3.6.1.4.1.35616.5.3.1.1.28.{{ $index }}'
+                    descr: Client {{ $index }} Temp
+                    index: 'lt2sClientLaserTemp.{{ $index }}'
+                -
+                    oid: lt2sTemperature
+                    value: lt2sTemperature
+                    num_oid: '.1.3.6.1.4.1.35616.5.2.1.{{ $index }}'
+                    descr: Chassis Temp
+                #HS variant
+                -
+                    oid: portEntry
+                    value: portClientLaserTemp
+                    num_oid: '.1.3.6.1.4.1.35616.2.1.1.12.1.17.{{ $index }}'
+                    descr: Client {{ $index }} Temp
+                    index: 'portClientLaserTemp.{{ $index }}'
+                    skip_values:
+                        -
+                            oid: portClientLaserTemp
+                            op: '='
+                            value: -1
+                -
+                    oid: systemTemperature
+                    value: systemTemperature
+                    num_oid: '.1.3.6.1.4.1.35616.2.1.1.1.0'
```

Comment:
You need to replace this line with: 
`num_oid: '.1.3.6.1.4.1.35616.2.1.1.1.{{ $index }}'`
There is only one entry so it won't change your display, but this ensures that the line is compatible with a table.

Reply:
Thanks for your instruction! 

I hope I fixed the nessessary stuff

Please advise for:
- lambdatrail is not realy an os, its more a product family, is this the right way?
- lt2-s and hs variante don't have much in common -  better have seperate definition/discovery.yaml ?
- the dbm low/high_limit don't have a divisor - should I switch to a php disovery?
- how can i set default low/high values for e.g. Temp (not delivered from snmp)

Reply:
@PipoCanaja anything I can optimize here?